eng/restore/docs.targets

```diff
@@ -4,43 +4,4 @@
     <PackageDownload Include="Microsoft.Private.Intellisense" Version="[$(MicrosoftPrivateIntellisenseVersion)]" />
   </ItemGroup>
 
-  <!-- the intellisense package doesn't use nuget conventions so we need to select manually -->
```

Comment:
Not needed anymore. We don't have localized folders in the nupkg anymore, and we are going to decide which xml to copy (from nupkg or from local build) inside intellisense.targets, not here.

We still need the first few lines that will download the package. Maybe we could move it somewhere else so this file gets deleted?

Reply:
Please move this PackageDownload item into the Build.proj file in the repo root, delete this file and remove the import to it.

Reply:
Done.